ircHandler.js

```diff
@@ -128,30 +128,19 @@ function checkForMissingStandups(config, callback) {
 
 exports.checkForMissingStandups = checkForMissingStandups;
 
+function remindMembers(message, missing, callback) {
+  var remind = function(member, callback) {
+    client.say(member, message);
+    callback();
+  };
 
-function announceEarlyReminder(config) {
-  checkForMissingStandups(config, function(err, missing) {
-    var msg = 'Standups are due soon. (' + missing.join(', ') + ')';
-    remindChannels(msg, function() {
-      console.log('Reminded channel that standups are due soon.');
-    });
-  });
-}
-
-function announceDueReminder(config) {
-  checkForMissingStandups(config, function(err, missing) {
-    var msg = 'Standups are due! (' + missing.join(', ') + ')';
-    remindChannels(msg, function() {
-      console.log('Reminded channel that standups are due now.');
-    });
-  });
+  async.forEach(missing, remind, callback);
 }
 
-function announceLateReminder(config) {
+function announceReminder(msg, config) {
   checkForMissingStandups(config, function(err, missing) {
-    var msg = 'Standups are late! (' + missing.join(', ') + ')';
-    remindChannels(msg, function() {
-      console.log('Reminded channels that standups are late.');
+    remindMembers('Standups are ' + msg, missing, function() {
```

Comment:
This is the output I get:
```
19:45:27 example | Standups are Standups are due!
19:45:27 example | Standups are Standups are late!
19:45:30 example | Standups are Standups are due soon.
19:45:33 example | Standups are Standups are due!    
```
You should remove `'Standups are '` from this line.